src/tz.cpp

```diff
@@ -277,8 +276,7 @@ namespace date
 
 using namespace detail;
 
-#if !USE_OS_TZDB
```

Comment:
Better to just leave this in

Reply:
Re-examining this change I can leave this in but it has knock on effects to a lower section of code responsible for fetching the TZDIR. That section can have its ifdef set to false, which does reduce over all line changes.